### PR TITLE
Prades publish of qt5

### PR DESCRIPTION
**Pull request by krunal-patel-fl4re**
This is similar to how we do recast publshing (as well as starvrsdk & cef).  
I can't run a PR on this as it is infrastructural; you'll have to merge to see if it works, and if not repeat & test.  In this case @mbarnes-sb please review and I will iterate on forcing publish to work.  
here is the job of interest: http://50.250.211.65:7070/view/ext/job/publish-recast/

### CODE REVIEW

File: publish_package.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# make sure we are inside a valid git dir
```
Comment (mbarnes-sb) [inline publish_package.sh]: needs `#!/bin/bash -ex` here or similar?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@sb/qt5",
+  "version": "0.0.0",
```
Comment (mbarnes-sb) [inline package.json]: not 0.1.0? how about tracking the Qt version major.minor version numbers?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "description": "fl4re qt5",
+  "repository": "https://github.com/fl4re/qt5",
+  "private": false,
+  "binary": {
+    "file": "{package_name}/{package_version}/qt5-{package_version}-{platform}-{arch}.tar.gz",
+    "host": "https://registry-node.starbreeze.com/-/releases",
+    "os": "win32",
```
Comment (mbarnes-sb) [inline package.json]: what about win64?

Comment (krunal-patel-fl4re) [inline package.json]: right, it's actually supposed to be only win64..will fix

File: package.json (hunk 2 above)
Comment (krunal-patel-fl4re) [inline package.json]: right, good idea.

File: publish_package.sh (hunk 1 above)
Comment (krunal-patel-fl4re) [inline publish_package.sh]: ok will do #!/bin/sh -xe

**Pull request merged**